### PR TITLE
Required changes for panda_moveit_config

### PR DESCRIPTION
**Pull request by rhaschke**
This merge commit gathers required changes for https://github.com/ros-planning/panda_moveit_config/issues/96. The individual PRs are listed in #193.

### CODE REVIEW

File: franka_gazebo/config/franka_hw_sim.yaml [hunk 1]
```diff
@@ -1,9 +1,9 @@
...
-m_ee:  0.76
+m_ee: 0.76
 franka_gripper:
-  type:    franka_gazebo/FrankaGripperSim
-  arm_id:  $(arg arm_id)
+  type: franka_gazebo/FrankaGripperSim
+  arm_id: $(arg arm_id)
 
   finger1:
     gains: { p: 100, i: 25, d: 20 }
```
Comment (rickstaa) [inline franka_gazebo/config/franka_hw_sim.yaml]: Just a quick reminder for ourselves that these PID gains need to be improved before this pull request is merged (see https://github.com/frankaemika/franka_ros/pull/173#issuecomment-938507534 for more information).

Comment (rickstaa) [reply franka_gazebo/config/franka_hw_sim.yaml]: Based on https://github.com/frankaemika/franka_ros/pull/173#issuecomment-964235123 this commit should be removed.

**Comment by rickstaa** [comment]
@rhaschke, it appears that many of the changes we've suggested in this pull request are already in some form within the `develop` branch. Considering this, would it be advisable to close this draft?

**Comment by rhaschke** [comment]
Closing, as most of the individual commits are merged upstream.

**Pull request closed without merging**